openedx_webhooks/templates/github_internal_cherrypick_comment.md.j2

```diff
@@ -0,0 +1,13 @@
+{% filter replace("\n", " ")|trim %}
+Well done, @{{ user }}! We have recently cut the {{ open_edx_release_name }} release candidate
```

Comment:
"release candidate" isn;t the right term here.  Maybe, "We've recently created the Hawthorn master branches"